app/code/Magento/Persistent/README.md

```diff
@@ -1,5 +1,63 @@
-Magento\Persistent module enables set customer a long-term cookie containing internal id (random hash - to exclude brute
+# Magento_Persistent module
+
+This module enables set customer a long-term cookie containing internal id (random hash - to exclude brute
 force) of persistent session. Persistent session data is kept in DB - so it's not deleted in some days and is kept for
 as much time as we need. DB session keeps customerId + some data from real customer session that we want to sync (e.g.
 num items in shopping cart). For registered customer this info is synced to persistent session if choose "Remember me"
 checkbox during first login.
+
+## Installation
+
+Before installing this module, note that the Magento_Persistent is dependent on the following modules:
+- `Magento_Checkout`
+- `Magento_PageCache`
+
+The Magento_Persistent module creates the `persistent_session` table in the database.
```

Comment:
Please add some information about the schema/column not being persistent (e.g. not having to be manually removed) when the module gets disabled and setup:upgrade is run.